metadata/config.yaml

```diff
@@ -40,34 +40,60 @@ DENEB_FORK_EPOCH: 0
 ELECTRA_FORK_VERSION: 0x60000910
 ELECTRA_FORK_EPOCH: 2048
 
+# Fulu
+FULU_FORK_VERSION: 0x70000910
+FULU_FORK_EPOCH: 50688
+
 # Time parameters
 # ---------------------------------------------------------------
-# 12 seconds
+# 12 seconds (*deprecated*)
 SECONDS_PER_SLOT: 12
+# 12000 milliseconds
+SLOT_DURATION_MS: 12000
 # 14 (estimate from Eth1 mainnet)
-SECONDS_PER_ETH1_BLOCK: 12
-# 2**8 (= 256) epochs ~27 hours
+SECONDS_PER_ETH1_BLOCK: 14
```

Comment:
Is this expected? This is as change unrelated to fusaka scheduling.

Reply:
Value comes from: https://github.com/ethereum/consensus-specs/blob/master/configs/mainnet.yaml#L78

Doubt its actually used by anyone?